modules.d/45net-lib/module-setup.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+check() {
+    return 255
+}
+
+depends() {
+    echo base
+    return 0
+}
+
+install() {
+    inst_script "$moddir/netroot.sh" "/sbin/netroot"
```

Comment:
`connman` is installing exactly the same `netroot.sh` script.

```
~/src/dracut/pr-2250> diff -s modules.d/45net-lib/netroot.sh modules.d/35connman/netroot.sh 
Files modules.d/45net-lib/netroot.sh and modules.d/35connman/netroot.sh are identical
```

After this PR, the following line can be removed from connman's `install()`, because `net-lib` is always installed as a dependency:

https://github.com/dracutdevs/dracut/blob/52351cfa049a9594d539e6a5337d591e8039ab80/modules.d/35connman/module-setup.sh#L28